crates/swc_common/src/plugin/serialized.rs

```diff
@@ -173,13 +173,11 @@ where
         .map_err(|_err| Error::msg("Failed to deserialize given ptr"))
 }
 
+/*
```

Comment:
@kdy1 somehow compilation fails as soon as downgrade rkyv related with deserialization logic with versionedserializable struct. This maybe partially related with the issues we're digging - tried to disable it for now until we can reliably figure out plugin workflows.

Reply:
Interesting... But yeah we should disable it for now.